package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "testplane-docs",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "docusaurus start",
+    "lint": "eslint . && prettier --check .",
+    "reformat": "eslint --fix . && prettier --write .",
+    "test": "npm run lint && npm run typecheck",
+    "typecheck": "tsc"
+  },
+  "dependencies": {
+    "@docusaurus/core": "3.1.1",
+    "@docusaurus/preset-classic": "3.1.1",
+    "@mdx-js/react": "3.0.1",
+    "clsx": "2.1.0",
```

Comment:
I dont see where you use it. Will be usefull in the future?

Reply:
Yeah, it's used everywhere in my react code to work with css class names. Keeping it.